utils/python/tests/list_tests

```diff
@@ -15,7 +16,11 @@ def list_tests_from():
                     print atest
                 for btest in atest._tests:
                     btestname = btest.__str__().split()
-                    print btestname[1][1:-1] + "." + btestname[0]
+                    test_classes.append(btestname[1][1:-1])
+	    # add this explicitly, not captured by the above
+	    test_classes.append("scripts_regression_tests.B_CheckCode")
```

Comment:
Looking at this more closely, why was B_CheckCode not captured? Also, looks like there may be an indentation error here.